### PR TITLE
chore: add annotation of locksript args source 

### PR DESCRIPTION
**Pull request by fpChan**

### CODE REVIEW

File: contracts/toCKB-lockscript/src/main.rs [hunk 1]
```diff
@@ -16,6 +16,9 @@ use core::result::Result;
 entry!(entry);
 default_alloc!();
 
+// total_size(4 byte) + offset(4 byte) * 3 + code_hash(32 byte) + hash_type(1 byte) + args_size(4 byte) + xchain_kind(1 byte) = 54 byte
+const MOLECULE_TYPESCRIPT_SIZE: usize = 54;
```
Comment (mkxbl) [inline contracts/toCKB-lockscript/src/main.rs:20]:
```suggestion
const TOCKB_LOCKSCRIPT_ARGS_LENGTH: usize = 54;
```

File: tests/src/toCKB_lockscript/mod.rs [hunk 2]
```diff
@@ -13,6 +13,8 @@ use tockb_types::basic;
 use tockb_types::generated::tockb_cell_data::ToCKBTypeArgs;
 
 const MAX_CYCLES: u64 = 10_000_000;
+// total_size(4 byte) + offset(4 byte) * 3 + code_hash(32 byte) + hash_type(1 byte) + args_size(4 byte) + xchain_kind(1 byte) = 54 byte
+const MOLECULE_TYPESCRIPT_SIZE: usize = 54;
```
Comment (mkxbl) [inline tests/src/toCKB_lockscript/mod.rs:17]:
```suggestion
const TOCKB_LOCKSCRIPT_ARGS_LENGTH: usize = 54;
```

File: contracts/toCKB-lockscript/src/main.rs [hunk 3]
```diff
@@ -16,6 +16,9 @@ use core::result::Result;
 entry!(entry);
 default_alloc!();
 
+// total_size(4 byte) + offset(4 byte) * 3 + code_hash(32 byte) + hash_type(1 byte) + args_size(4 byte) + xchain_kind(1 byte) = 54 byte
```
Comment (mkxbl) [inline contracts/toCKB-lockscript/src/main.rs:19]:
```suggestion
// first 54 bytes of toCKB typescript molecule bytes: 
// total_size(4 byte) + offset(4 byte) * 3 + code_hash(32 byte) + hash_type(1 byte) + args_size(4 byte) + xchain_kind(1 byte) = 54 byte
```

File: tests/src/toCKB_lockscript/mod.rs [hunk 4]
```diff
@@ -13,6 +13,8 @@ use tockb_types::basic;
 use tockb_types::generated::tockb_cell_data::ToCKBTypeArgs;
 
 const MAX_CYCLES: u64 = 10_000_000;
+// total_size(4 byte) + offset(4 byte) * 3 + code_hash(32 byte) + hash_type(1 byte) + args_size(4 byte) + xchain_kind(1 byte) = 54 byte
```
Comment (mkxbl) [inline tests/src/toCKB_lockscript/mod.rs:16]:
```suggestion
// first 54 bytes of toCKB typescript molecule bytes: 
// total_size(4 byte) + offset(4 byte) * 3 + code_hash(32 byte) + hash_type(1 byte) + args_size(4 byte) + xchain_kind(1 byte) = 54 byte
```

File: tests/src/toCKB_lockscript/mod.rs (hunk 2 above)
Comment (fpChan) [reply tests/src/toCKB_lockscript/mod.rs]: done

**Pull request merged**